cmd/skaffold/app/cmd/build.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/flags"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/config"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner"
-	latestV1 "github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest/v1"
+	latestV2 "github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest/v2"
```

Comment:
we can alias the `../schema/v1` import statements on `main` branch and `../schema/v2` on the `v2` branch, to something like `schema`. Then we'd get far fewer diffs, and it'd probably be easier to review. Did we already discuss this approach?

Reply:
I think this might have been mentioned in the past but hasn't been discussed in a while. It's a good idea imo

Reply:
wouldn't all of these still show the change from 
`latest "github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest/v1"`
-> 
`latest "github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest/v2"`
?

Unless we also went and removed the unnecessary `v1` or `v2` package so that each would just have `pkg/skaffold/schema/latest` with no subpackages

Reply:
this is done.